### ISSUE TITLE
underscores in labels are no longer supported

### ISSUE THREAD

**Issue by alistair-broomhead** [knsv/mermaid#915]
**Describe the bug**
Upon upgrading from 8.2.3 to 8.2.4 you can no longer use underscores in labels. The results in links to the live editor created last week now leading to an error page

**To Reproduce**
[Create a graph using the following](https://mermaidjs.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ3JhcGggTFJcblxuZm9vW215X2xpYi5mb29dXG5iYXJbbXlfbGliLmJhcl1cbmJheltteV9saWIucXV4LmJhel1cblxuZm9vIC0tPiBiYXJcbmJhciAtLT4gYmF6IiwibWVybWFpZCI6eyJ0aGVtZSI6ImRlZmF1bHQifX0/error/UGFyc2UgZXJyb3Igb24gbGluZSAzOgpncmFwaCBMUmZvb1tteV9saWIuZm9vXWJhcltteV9saWIKLS0tLS0tLS0tLS0tLS1eCkV4cGVjdGluZyAnU0VNSScsICdORVdMSU5FJywgJ1NQQUNFJywgJ0VPRicsICdHUkFQSCcsICdESVInLCAnVEFHRU5EJywgJ1RBR1NUQVJUJywgJ1VQJywgJ0RPV04nLCAnc3ViZ3JhcGgnLCAnU1FTJywgJ1NRRScsICdlbmQnLCAnUEUnLCAnLSknLCAnRElBTU9ORF9TVE9QJywgJ1RSQVBFTkQnLCAnSU5WVFJBUEVORCcsICctLScsICdBUlJPV19QT0lOVCcsICdBUlJPV19DSVJDTEUnLCAnQVJST1dfQ1JPU1MnLCAnQVJST1dfT1BFTicsICdET1RURURfQVJST1dfUE9JTlQnLCAnRE9UVEVEX0FSUk9XX0NJUkNMRScsICdET1RURURfQVJST1dfQ1JPU1MnLCAnRE9UVEVEX0FSUk9XX09QRU4nLCAnPT0nLCAnVEhJQ0tfQVJST1dfUE9JTlQnLCAnVEhJQ0tfQVJST1dfQ0lSQ0xFJywgJ1RISUNLX0FSUk9XX0NST1NTJywgJ1RISUNLX0FSUk9XX09QRU4nLCAnUElQRScsICdTVFlMRScsICdMSU5LU1RZTEUnLCAnQ0xBU1NERUYnLCAnQ0xBU1MnLCAnQ0xJQ0snLCAnREVGQVVMVCcsICdQQ1QnLCAnTlVNJywgJ0NPTU1BJywgJ0FMUEhBJywgJ0NPTE9OJywgJ01JTlVTJywgJ0JSS1QnLCAnRE9UJywgJ1BVTkNUVUFUSU9OJywgJ1VOSUNPREVfVEVYVCcsICdQTFVTJywgJ0VRVUFMUycsICdNVUxUJywgZ290ICdVTkRFUlNDT1JFJw):
```
graph LR

foo[my_lib.foo]
bar[my_lib.bar]
baz[my_lib.qux.baz]

foo --> bar
bar --> baz
```

**Expected behavior**
![Screenshot 2019-08-28 at 11 06 56](https://user-images.githubusercontent.com/1235814/63846579-10db6780-c9c7-11e9-8d54-935b5acce87c.png)

**Screenshots**
![Screenshot 2019-08-28 at 11 08 50](https://user-images.githubusercontent.com/1235814/63846727-50a24f00-c9c7-11e9-913a-94b5f89d81ac.png)

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser Chrome
 - Version 75

**Additional context**
The reproduction screenshots were taken using local checkouts of the live editor repo, but 
the issue is live at [mermaidjs.github.io/mermaid-live-editor](https://mermaidjs.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ3JhcGggTFJcblxuZm9vW215X2xpYi5mb29dXG5iYXJbbXlfbGliLmJhcl1cbmJheltteV9saWIucXV4LmJhel1cblxuZm9vIC0tPiBiYXJcbmJhciAtLT4gYmF6IiwibWVybWFpZCI6eyJ0aGVtZSI6ImRlZmF1bHQifX0/error/UGFyc2UgZXJyb3Igb24gbGluZSAzOgpncmFwaCBMUmZvb1tteV9saWIuZm9vXWJhcltteV9saWIKLS0tLS0tLS0tLS0tLS1eCkV4cGVjdGluZyAnU0VNSScsICdORVdMSU5FJywgJ1NQQUNFJywgJ0VPRicsICdHUkFQSCcsICdESVInLCAnVEFHRU5EJywgJ1RBR1NUQVJUJywgJ1VQJywgJ0RPV04nLCAnc3ViZ3JhcGgnLCAnU1FTJywgJ1NRRScsICdlbmQnLCAnUEUnLCAnLSknLCAnRElBTU9ORF9TVE9QJywgJ1RSQVBFTkQnLCAnSU5WVFJBUEVORCcsICctLScsICdBUlJPV19QT0lOVCcsICdBUlJPV19DSVJDTEUnLCAnQVJST1dfQ1JPU1MnLCAnQVJST1dfT1BFTicsICdET1RURURfQVJST1dfUE9JTlQnLCAnRE9UVEVEX0FSUk9XX0NJUkNMRScsICdET1RURURfQVJST1dfQ1JPU1MnLCAnRE9UVEVEX0FSUk9XX09QRU4nLCAnPT0nLCAnVEhJQ0tfQVJST1dfUE9JTlQnLCAnVEhJQ0tfQVJST1dfQ0lSQ0xFJywgJ1RISUNLX0FSUk9XX0NST1NTJywgJ1RISUNLX0FSUk9XX09QRU4nLCAnUElQRScsICdTVFlMRScsICdMSU5LU1RZTEUnLCAnQ0xBU1NERUYnLCAnQ0xBU1MnLCAnQ0xJQ0snLCAnREVGQVVMVCcsICdQQ1QnLCAnTlVNJywgJ0NPTU1BJywgJ0FMUEhBJywgJ0NPTE9OJywgJ01JTlVTJywgJ0JSS1QnLCAnRE9UJywgJ1BVTkNUVUFUSU9OJywgJ1VOSUNPREVfVEVYVCcsICdQTFVTJywgJ0VRVUFMUycsICdNVUxUJywgZ290ICdVTkRFUlNDT1JFJw)

**Comment by JonMidhir** [comment]
Confirming that we've experienced this as well after upgrading from 8.2.3 to 8.2.4.

A workaround is to replace underscores (`_`) with colons (`:`) however this potentially requires a project-wide update.

**Comment by knsv** [comment]
Will fix it in 8.2.6

**Comment by JonMidhir** [comment]
@knsv Actually I was mistaken, our error is slightly different from the OP. We get an error when using underscores in class names in 8.2.4. In 8.2.5 it looks like this goes away but occurs instead when using the class name as the first argument to `click`.

![image](https://user-images.githubusercontent.com/39089/63874514-922bfd80-c9b9-11e9-9200-d47062434d00.png)

Should I create a separate issue for this or do you think these are related and will both be fixed in 8.2.6?

**Comment by alistair-broomhead** [comment]
@knsv Thanks, and kudos on such a quick response/turn-around

**Comment by knsv** [comment]
Released in 8.2.6
